src/seedu/addressbook/AddressBook.java

```diff
@@ -202,12 +202,7 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
-            String userCommand = getUserInput();
-            echoUserCommand(userCommand);
-            String feedback = executeCommand(userCommand);
-            showResultToUser(feedback);
-        }
+        ifTrue();
```

Comment:
do take note we name methods. Method names should be descriptive of what it does. (: